package.json

```diff
@@ -35,6 +35,7 @@
     "resig-class": "^1.0.0"
   },
   "dependencies": {
+    "grunt-cli": "^1.3.2",
```

Comment:
Ah, i think this may need to be in "dev-dependencies" so that it only gets included for developers, not for deployment? not a big deal but good practice. Thank you! https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file

Reply:
Oh that's interesting. I installed it and didn't realize it would get added in there. I'll move it to dev-dependencies!